projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -3,96 +3,232 @@ import { getRedirectUrl } from '@automattic/jetpack-components';
-import InspectorNotice from '../../shared/components/inspector-notice';
 import { META_NAME_FOR_POST_LEVEL_ACCESS_SETTINGS } from './constants';
 import { getPaidPlanLink } from './utils';
 
 import './settings.scss';
 
 export const accessOptions = {
 	everybody: {
-		string: 'everybody',
+		key: 'everybody',
 		label: __( 'Everybody', 'jetpack' ),
 		info: __( 'Visible to everyone.', 'jetpack' ),
 	},
 	subscribers: {
-		string: 'subscribers',
+		key: 'subscribers',
 		label: __( 'All subscribers', 'jetpack' ),
-		info: __( 'Visible to everyone that subscribes to your site.', 'jetpack' ),
+		info: __( 'Anyone subscribed to your newsletter.', 'jetpack' ),
 	},
 	paid_subscribers: {
-		string: 'paid_subscribers',
+		key: 'paid_subscribers',
 		label: __( 'Paid subscribers', 'jetpack' ),
-		info: __( 'Visible to everyone that purchases a paid plan on your site.', 'jetpack' ),
+		info: __( 'Only for paid subscribers.', 'jetpack' ),
 	},
 };
 
-function getReachForAccessLevel( key, emailSubscribers, paidSubscribers, socialFollowers ) {
+function Link( { slug, children } ) {
+	return (
+		<a
+			target="_blank"
+			rel="noopener noreferrer"
+			href={ getRedirectUrl( slug ) }
+			className="jetpack-newsletter-link"
+		>
+			{ children }
+		</a>
+	);
+}
+
+function getReachForAccessLevelKey(
+	accessLevelKey,
+	emailSubscribers,
+	paidSubscribers,
+	socialFollowers
+) {
 	if ( emailSubscribers === null || paidSubscribers === null || socialFollowers === null ) {
-		return '';
+		return 0;
 	}
 
-	switch ( accessOptions[ key ] ) {
-		case accessOptions.everybody:
-			return '(' + ( emailSubscribers + socialFollowers ) + '+)';
-		case accessOptions.subscribers:
-			return '(' + emailSubscribers + ')';
-		case accessOptions.paid_subscribers:
-			return '(' + paidSubscribers + ')';
+	switch ( accessOptions[ accessLevelKey ].key ) {
+		case accessOptions.everybody.key:
+			return emailSubscribers;
+		case accessOptions.subscribers.key:
+			return emailSubscribers;
+		case accessOptions.paid_subscribers.key:
+			return paidSubscribers;
 		default:
+			return 0;
 	}
 }
 
-export function MisconfigurationWarning( { accessLevel } ) {
+function NewsletterLearnMore() {
 	return (
-		<div className="jetpack-subscribe-notice-misconfiguration warning">
-			{ sprintf(
-				/* translators: %1$s: visibility label for the newsletter, %2$s: label for setting "everybody". this is a warning in the newsletter when posts have a private or password-protected visibility */
+		<small className="jetpack-newsletter-learn-more">
+			{ createInterpolateElement(
 				__(
-					'Private or password-protected posts cannot be assigned a newsletter setting of "%1$s". Please update the setting to "%2$s", or update the post visibility setting.',
+					'Restrict your post to subscribers. <learnMoreLink>Learn more.</learnMoreLink>',
 					'jetpack'
 				),
-				accessOptions[ accessLevel ].label,
-				accessOptions.everybody.label
+				{
+					learnMoreLink: (
+						<Link
```

Comment:
Why not just `ExternalLink` from `@wordpress/components`?

Reply:
I went with this approach because the `ExternalLink` includes an icon that the designs doesn't have.  I felt this was an easier approach. Open to changing it!

Reply:
@crisbusquets what do you think about having the "external link" icon at least in some of the links that open in new tabs? It's not a strict convention in Gutenberg but used a lot. Makes it less scary to click while they're editing the content, since they know it opens a new tab. :-)

Example:

<img width="290" alt="Screenshot 2023-05-04 at 11 37 57" src="https://user-images.githubusercontent.com/87168/236153490-1d5bc11a-b405-44fc-8db1-685756ddbfff.png">

Reply:
@simison From an accessibility point of view, we should always identify links that open on new tabs/windows. Having said that, I'm concerned that we would have too many icons. Could you try this to see how it would look?

Reply:
I tried. It looks pretty bad.